tools/src/main/java/org/apache/kafka/trogdor/workload/ConsumeBenchSpec.java

```diff
@@ -63,13 +87,22 @@ public ConsumeBenchSpec(@JsonProperty("startMs") long startMs,
         this.commonClientConf = configOrEmptyMap(commonClientConf);
         this.adminClientConf = configOrEmptyMap(adminClientConf);
         this.activeTopics = activeTopics == null ? TopicsSpec.EMPTY : activeTopics.immutableCopy();
+        this.consumerGroup = consumerGroup;
```

Comment:
Should be `consumerGroup == null ? "" : consumerGroup` to match the other entries.  We don't use nulls in JSON